src/utils/ignore-types.ts

```diff
@@ -132,7 +132,7 @@ function replaceRegex(text: string, placeholder: string, regex: RegExp): IgnoreR
  * @return {string[]} The regular markdown links replaced
  */
 function replaceMarkdownLinks(text: string, regularLinkPlaceholder: string): IgnoreResults {
-  const positions: Position[] = getPositions(MDAstTypes.Link, text);
+  const positions: Position[] = getPositions([MDAstTypes.Link, MDAstTypes.Image], text);
```

Comment:
Why add `image` to this list as opposed to adding it to the ignore types list for the function?

Reply:
@pjkaufman no reason :)

Reply:
I will fix it